### PR TITLE
Enhance tabs and editor

### PR DESCRIPTION
**Pull request by MaxGraey**
Editor (before):
editor-before

**Editor (after):**
after

### CODE REVIEW

**Comment by dcodeIO** [comment]
image

**Comment by MaxGraey** [comment]
@dcodeIO good catch! fixed

**Comment by dcodeIO** [comment]
Now it fits exactly :)

File: src/.vuepress/public/editor.html [hunk 1]
```diff
@@ -398,7 +398,16 @@ <h5>Documentation</h5>
...
-      languages.setLanguageConfiguration('webassembly', wasmLanguageConfig)
+      languages.setLanguageConfiguration('webassembly', {
+        ...wasmLanguageConfig,
+        brackets: [['(', ')'], ['if', 'end'], ['loop', 'end'], ['block', 'end']],
+        surroundingPairs: [
+          { open: '(', close: ')' },
+          { open: 'if', close: 'end' },
+          { open: 'loop', close: 'end' },
+          { open: 'block', close: 'end' }
+        ]
```
Comment (dcodeIO) [inline src/.vuepress/public/editor.html]: Looks like these could go into the [config file](https://github.com/AssemblyScript/website/blob/main/src/.vuepress/public/scripts/webassembly-language.js) itself. Includes a few of these already

Comment (MaxGraey) [reply src/.vuepress/public/editor.html]: already moved it to config =)

Comment (MaxGraey) [reply src/.vuepress/public/editor.html]: here: https://github.com/AssemblyScript/website/pull/108/commits/1c90a06897c7ebc5ddc11942994de4e03eb4284a

**Review by dcodeIO — APPROVED** [review]
Tested it again locally, works like a charm :)

**Pull request merged**